js/components/CategoryCard.jsx

```diff
@@ -0,0 +1,55 @@
+var React = require('react');
+
+var Link = require('react-router').Link;
+
+var MiniInvestButton = require('./MiniInvestButton.jsx');
+
+var CategoryCard = React.createClass({
```

Comment:
Some of these cards bleed over the edge of the box. I also think this could be styled a little more, but I'm okay with pushing an iteration 0. But, I do think the users bleeding over the boxes is pretty strange.